dev/run-tests.py

```diff
@@ -112,6 +112,7 @@ def determine_modules_to_test(changed_modules):
     ['graphx', 'examples']
     >>> x = [x.name for x in determine_modules_to_test([modules.sql])]
     >>> x # doctest: +NORMALIZE_WHITESPACE
+    ...   # doctest: +SKIP
     ['sql', 'avro', 'hive', 'mllib', 'sql-kafka-0-10', 'examples', 'hive-thriftserver',
```

Comment:
l manually tested with `HADOOP_PROFILE=hadoop3.2 python -m doctest run-tests.py` and `python -m doctest run-tests.py`